src/harvester/wasser_de.rs

```diff
@@ -71,13 +73,16 @@ async fn translate_dataset(dir: &Dir, source: &Source, document: Document) -> Re
         .or(document.auto_teaser_text)
         .unwrap_or_default();
 
+    let date: Date = Date::from_ordinal_date(document.dct_issued.unwrap_or(0), 1).unwrap();
```

Comment:
To map the optional value, use [`Option::map`](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.map), i.e.

```rust
let date = document.year_issued.map(|year_issued| Date::from_ordinal_date(year_issued, 1));
```

(The type ascription on the left-hand side is also not necessary here.)